### ISSUE TITLE
Feature Request - fix color palette for color-blind individuals

### ISSUE THREAD

**Issue by keg34** [ArctosDB/arctos#4058]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
The Fields list on the [Bulkloader page ](https://handbook.arctosdb.org/documentation/bulkloader.html) in the Arctos Handbook uses green and red to denote required vs. not-required fields.  My red-green colorblind students can't distinguish those colors, so this is not helpful to them.

**Describe what you're trying to accomplish** 
I want my students to be able to use the list as intended.

**Describe the solution you'd like**
Change one of the colors.  Say, change green to blue.

**Comment by keg34** [comment]
Here's what they say: "The best 3 to me are the maroon #800000; yellow #ffff00; and a light grey #33cccc"

They also suggested that it would be cool if there were a way to have users define color palette preferences, which could be tailored to different types of color-vision impairment (there are several).  We agreed that this probably would be a big challenge on the programming end, so not probably realistic, but I figured I would at least mention it.

Thanks for working on this, Dusty!

**Comment by dustymc** [comment]
I did this:

```


.blrequired {
color: #800000;
}

.blcondreq {
  color: ##cccc00;
}

.bloptional {
  color: #33cccc;
}
```

I couldn't see that yellow so I changed the optional marker; let me know if that's OK.

We have very little control over the autogenerated github help pages. Core Arctos could swap CSS files by request, but they're probably better off getting a browser extension which would work anywhere (and/or helping us pick default colors...).

FWIW I'm hoping to retire that page for something generated by https://arctos.database.museum/tblbrowse.cfm?tbl=bulkloader, and the requirements will just be in the text when/if that happens.
